From b78617c21f40f07b09e315f466cea3f05fc3ad16 Mon Sep 17 00:00:00 2001
From: LeonardHerbst <leonard.herbst@tu-dresden.de>
Date: Fri, 24 Apr 2026 13:09:45 +0200
Subject: [PATCH] tmp

---
 ports/riot/mpconfigport.h | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/ports/riot/mpconfigport.h b/ports/riot/mpconfigport.h
index b36dd7234..30e9e0789 100644
--- a/ports/riot/mpconfigport.h
+++ b/ports/riot/mpconfigport.h
@@ -33,6 +33,10 @@
 #define MICROPY_HEAP_SIZE (16 * 1024)
 #endif
 
+#ifndef MICROPY_LONGINT_IMPL
+#define MICROPY_LONGINT_IMPL (MICROPY_LONGINT_IMPL_LONGLONG)
+#endif
+
 #define MICROPY_ENABLE_SOURCE_LINE      (1)
 #define MICROPY_STACK_CHECK             (1)
 #define MICROPY_ENABLE_GC               (1)
@@ -81,7 +85,6 @@
 #define MICROPY_PY_RIOT                 (1)
 #define MICROPY_PY_XTIMER               (1)
 #define MICROPY_PY_SYS_MODULES          (1)
-#define MICROPY_LONGINT_IMPL            (MICROPY_LONGINT_IMPL_LONGLONG)
 #define MICROPY_FLOAT_IMPL              (MICROPY_FLOAT_IMPL_FLOAT)
 #define MICROPY_PY_BUILTINS_COMPLEX     (0)
 
-- 
2.51.0

